packages/openapi-ts/src/generate/services.ts

```diff
@@ -1,8 +1,7 @@
+import type { ClassElement, FunctionParameter } from '../compiler';
```

Comment:
shouldn't this pull Comments and Node too?

Reply:
Hmm, I'm not sure why eslint picked this option - I ran . I agree it looks a bit weird.

I will try to fix it manually.